frontend/alanda/libs/common/src/lib/components/task-table/task-table.component.ts

```diff
@@ -337,6 +337,9 @@ export class AlandaTaskTableComponent {
   );
 
   delegateTask$ = this.delegateTaskEvent$.pipe(
+    tap(() => {
+      this.state.set({ showDelegateDialog: true });
+    }),
     switchMap((data: AlandaTaskListData) => {
```

Comment:
bei dem map in der funktion unten solltest du dann nicht mehr den showDelegateDialog: true nochmal setzten ich würde das unten rausnehmen